feature/navigation/drawer/src/debug/kotlin/app/k9mail/feature/navigation/drawer/ui/DrawerContentPreview.kt

```diff
@@ -99,3 +106,148 @@ internal fun DrawerContentWithSelectedUnifiedFolderPreview() {
         )
     }
 }
+
+@Composable
+@Preview(showBackground = true)
+internal fun DrawerContentSingleAccountPreview() {
+    val displayFolders = createDisplayFolderList(hasUnifiedFolder = false)
+
+    PreviewWithTheme {
+        DrawerContent(
+            state = DrawerContract.State(
+                accounts = persistentListOf(
+                    DISPLAY_ACCOUNT,
+                ),
+                selectedAccountId = DISPLAY_ACCOUNT.id,
+                folders = displayFolders,
+                selectedFolderId = displayFolders[0].id,
+                showAccountSelector = false,
+            ),
+            onEvent = {},
+        )
+    }
+}
+
+@Composable
+@Preview(showBackground = true)
+internal fun DrawerContentSingleAccountWithAccountSelectionPreview() {
+    val displayFolders = createDisplayFolderList(hasUnifiedFolder = false)
+
+    PreviewWithTheme {
+        DrawerContent(
+            state = DrawerContract.State(
```

Comment:
Optional: Since this preview is about the account selection area specifically, I think it would be better to explicitly include `showAccountSelector = true`. That way this preview is independent of the default value in `DrawerContent.State`.

The same point applies to a few of the other previews.